app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Hello World React Rails</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
```

Comment:
- [ ] When starting the app, I am getting an error, please write in comments if any other extra steps are required to start the app, maybe Im missing anything, I have:
- installed database, 
- run the migrations,
- run `bundle install` 
- run `npm i`, 
- then I run `./bin/dev` to open the app,

 Error message: 
<img width="1165" alt="Screenshot 2022-08-15 at 22 27 52" src="https://user-images.githubusercontent.com/77038610/184693881-224128e4-0f0d-4938-a717-6b00b8a14612.png">